polyglot-kotlin/README.md

```diff
@@ -1,57 +1,106 @@
 # Polyglot Kotlin  
-Here is the code:
+
+## Overview
+
+- Supports adhoc task execution via Kotlin lambdas and external scripts.
+- Supports the complete Maven model.
+- Preserves well-known Maven idioms with a Kotlin flavor, allowing easy adoption.
+- Supports Xpp3DOM (XML) configuration using idiomatic Kotlin.
+- Provides idiomatic Kotlin extensions that improve readability and reduce lines of code.
+
+## Kotlin POM Example
+
 ```kotlin
-project {
-    name = "Polyglot :: Kotlin"
```

Comment:
So is this longer and explicit (and more consistent setup with the original pom) no longer supported? If so why?

Reply:
That remains supported. Under `polyglot-kotlin/src/test/resources/unit-tests/name/`, there are three different ways the name can be supplied.

Reply:
Great